### PR TITLE
Bump libs to address security vulnerabilities

### PR DESCRIPTION
**Pull request by benjben**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -149,18 +153,24 @@ jobs:
...
         command: monitor
       env:
         SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+    - name: Run Snyk to check for vulnerabilities (-rc)
+      uses: snyk/actions/docker@master
+      if: ${{ ! matrix.final_release }}
+      with:
+        image: "${{ steps.packageName.outputs.package_name }}:${{ steps.ver.outputs.tag }}-distroless"
+        args: "--app-vulns"
+        command: monitor
```
Comment (istreeter) [inline .github/workflows/release.yml:173]: Is the intention for this to go to our non-prod dashboard?

Comment (benjben) [reply .github/workflows/release.yml:173]: Yeah, or just display it in the Github action, but that [failed](https://github.com/snowplow/enrich/actions/runs/7668099290/job/20899268003)

Comment (istreeter) [reply .github/workflows/release.yml]: How does it work though?  I assumed our `SNYK_TOKEN` is scoped only for one organization?  Or is that a wrong assumption -- maybe it has permissions to push to both organizations?

Should we specify the `--org=something` so we explicitly tell it where to send the results?

Comment (benjben) [reply .github/workflows/release.yml]: You're right, it is scoped for `data-processing-new`. I removed it

`for queue in kinesis pubsub kafka nsq; do sbt "project ${queue}Distroless; set version := \"latest\"; Docker / publishLocal"; snyk container test --app-vulns snowplow/snowplow-enrich-$queue:latest-distroless; done` FTW!

**Review by colmsnowplow — APPROVED** [review]
LGTM

**Pull request merged**